src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -124,41 +125,44 @@ export function LocationDetails({
           <>
             <div className={styles.measurements}>
               <div className={styles.singleDetail}>
-                <p className={styles.title}>Measurements</p>
+                <p className={styles.title}>{t('measurements')}</p>
               </div>
             </div>
             <div className={styles.measurements}>
               <div className={styles.singleDetail}>
-                <p className={styles.title}>Date</p>
-                <div className={styles.value}>
-                  {location.measurements?.height}
-                  {t('m')}
-                </div>
+                <p className={styles.title}>{t('date')}</p>
+                {location.history?.map((h, index) => (
+                  <div className={styles.value} key={index}>
+                    {moment(h?.created).format('DD-MM-YYYY')}
```

Comment:
We already have some date formatting at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/88ca6a15bc349ab9d59261db47718e42f997de4d/src/utils/countryCurrency/getFormattedDate.ts#L5